From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: slitaz-cn <teasiu@163.com>
Date: Sun, 25 Apr 2021 10:41:55 +0800
Subject: [PATCH] add ftp-wan port

Signed-off-by: slitaz-cn <teasiu@163.com>
---
 net/vsftpd/files/vsftpd.conf | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/net/vsftpd/files/vsftpd.conf b/net/vsftpd/files/vsftpd.conf
index e06ac3543..498ce3e93 100644
--- a/net/vsftpd/files/vsftpd.conf
+++ b/net/vsftpd/files/vsftpd.conf
@@ -27,4 +27,6 @@ session_support=NO
 #ssl_sslv3=NO
 #rsa_cert_file=/etc/vsftpd/vsftpd_cert.pem
 #rsa_private_key_file=/etc/vsftpd/vsftpd_privkey.pem
-
+pasv_enable=YES
+pasv_min_port=10090
+pasv_max_port=10100
-- 
2.20.1

